lib/features/guide/widgets/about_us/models/link_data.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:collection/collection.dart';
+
+import '../../../../../config/ui_config.dart';
+
+class LinkData {
+  String iconUrl;
+  String? url;
+
+  static const _icons = {
+    "facebook": "assets/icons/ic_fb.png",
+    "instagram": "assets/icons/ic_insta.png",
+    "linkedin": "assets/icons/ic_linkedin.png",
+    "github": "assets/icons/ic_github.png",
+    "mailto:": "assets/icons/ic_email.png",
+    "pwr.edu.pl": "assets/icons/ic_web.png",
+  };
+
+  LinkData({
```

Comment:
so this whole file is basically repeated version of `lib\features\study_circle_details\smth...` 

it's probably good idea to make it a global util/model and use it anywhere in the app we need it

#### proposition
my propo would maybe be:
1. `lib\config\url_icons.dart`:
```dart
abstract class UrlIconsConfig {
 static const icons = {
     "facebook": "assets/icons/ic_fb.png",
    "instagram": "assets/icons/ic_insta.png",
    "linkedin": "assets/icons/ic_linkedin.png",
    "github": "assets/icons/ic_github.png",
    "mailto:": "assets/icons/ic_email.png",
  };
}
```

4.  combine `ContactSectionData` and `LinkData`  into  `lib/utils/urls_icons.dart`:

```dart
@immutable
class UrlIconsUtilModel {
  final String iconUrl;
  final String? text;
  final String? url;

  UrlIconsUtilModel({
    String? text,
    this.url,
  })  : iconUrl = _determineIcon(url),
        text = text ?? url;

  static String _determineIcon(String? url) {
    return url != null
        ? UrlIconsConfig.icons.entries
                .firstWhereOrNull(
                  (e) => url.contains(e.key),
                )
                ?.value ??
            DetailsScreenConfig.defaultIconUrl
        : DetailsScreenConfig.defaultIconUrl;
  }
}
```

5. delete both your `LinkData` and original `ContactSectionData` and use our new util model everywhere